src/app/components/SearchContainer/SearchContainer.jsx

```diff
@@ -93,36 +93,38 @@ class SearchContainer extends React.Component {
       <main id="mainContent">
         <div className="nypl-full-width-wrapper">
           <div className="nypl-page-header">
-            <Breadcrumbs
-              links={[
-                {
-                  href: `/search?q=${selectedQuery}&field=${selectedField}`,
-                  text: 'Search Results',
-                },
-              ]}
-              pageType={pageType}
-              onClickHandler={handleReset}
-            />
+            <div className="search-wrapper">
+              <Breadcrumbs
+                links={[
+                  {
+                    href: `/search?q=${selectedQuery}&field=${selectedField}`,
+                    text: 'Search Results',
+                  },
+                ]}
+                pageType={pageType}
+                onClickHandler={handleReset}
+              />
+            </div>
           </div>
           <div
             role="search"
             aria-label="ResearchNow"
           >
             {(!searchResults || _isEmpty(searchResults)) && (
-              <div className="nypl-row">
-                <div className="nypl-column-full">
-                  <h1 className="nypl-heading">ResearchNow</h1>
-                  <div id="tagline">Search the world&apos;s research collections and more for digital books you can use right now.</div>
-                </div>
+              <div className="nypl-row search-wrapper">
+                <h1 className="nypl-heading">ResearchNow</h1>
+                <div id="tagline">Search the world&apos;s research collections and more for digital books you can use right now.</div>
               </div>
             )}
             <div className="wrapper">
-              <SearchForm
-                searchQuery={selectedQuery}
-                searchField={selectedField}
-                history={history}
-                {...this.boundActions}
-              />
+              <div className="search-wrapper">
```

Comment:
This div is a `search-wrapper` but the other two divs above are not wrapping any search functionality or divs. A better name than `search-wrapper` would be ideal since it is wrapping the breadcrumbs and h1 and no search there.

Reply:
Changed name to header-wrapper